pkg/operator/controllers/genevalogging/genevalogging.go

```diff
@@ -21,12 +21,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
-const (
-	GenevaCertName = "gcscert.pem"
-	GenevaKeyName  = "gcskey.pem"
-)
-
-func (g *GenevaloggingReconciler) securityContextConstraints(ctx context.Context, name, serviceAccountName string) (*securityv1.SecurityContextConstraints, error) {
+func (g *Reconciler) securityContextConstraints(ctx context.Context, name, serviceAccountName string) (*securityv1.SecurityContextConstraints, error) {
```

Comment:
Should we standardize on the `g` being an `r` like all the other `*Reconciler`(s)

Reply:
Sure